include/votca/tools/vec.h

```diff
@@ -104,7 +105,22 @@ class vec {
      */
     const double &getZ() const { return _z; }
     
-    
+     /**
+       * \brief Compare floating point values of two vectors
+       *
+       * The floating point values are compared to within a tolerance which
+       * is specified as the third parameter.
+       *
+       * @param[in] v - vector to be compared
+       * @param[in] tol - tolerance 
+       * @return bool - return true if within tolerance and false if not
+       */
+      bool isClose(const vec& v, double tol) const {
+        for (size_t i = 0; i < 3; ++i){
+          if (!isApproximatelyEqual(xyz[i], v.xyz[i],tol)) return false;
+        }
+        return true;
+}
```

Comment:
You can the inline command to this function as well.

Reply:
Make tol a const double as well .

Reply:
yeah inlining, I would prefer to have .cc files soon.